src/Dfc.CourseDirectory.Web/ViewComponents/Dashboard/Default.cshtml

```diff
@@ -127,14 +127,17 @@ else
                         </div>
                         break;
                     case ProviderType.Apprenticeship:
-                        <div class="govuk-grid-row govuk-!-margin-bottom-6 homepage-item--container">
-                            <div class="govuk-grid-column-one-half homepage-item ">
-                                @await Html.PartialAsync("/ViewComponents/Dashboard/_ApprenticeshipsCardPartial.cshtml")
-                            </div>
-                            <div class="govuk-grid-column-one-half homepage-item ">
-                                @await Html.PartialAsync("/ViewComponents/Dashboard/_VenuesCardPartial.cshtml")
-                            </div>
-                        </div>
+        <div class="govuk-grid-row govuk-!-margin-bottom-6 homepage-item--container">
+            @if(Model.ProviderQACurrentStatus == Dfc.CourseDirectory.WebV2.Models.ApprenticeshipQAStatus.Passed)
```

Comment:
Need an additional check here - if the QA feature is *not* enabled then should show this UI also

Reply:
condition added